tests/tests/test_deactivate_expired_reasons.py

```diff
@@ -74,3 +74,21 @@ def test_command_should_not_anonymize_customer_if_expiring_reason_is_not_related
         self.assertAnonymizedDataNotExists(self.customer, 'first_name')
         assert_not_equal(self.customer._anonymize_obj(), CUSTOMER__FIRST_NAME)
         self.assertAnonymizedDataExists(self.customer, 'first_name')
+
+    def test_facebook_purpose_should_anonymize_customer_with_facebook_id(self):
```

Comment:
Wrong file for these tests? This one is intended for the reason expiration command.

Reply:
you are true I will move it